include/libcyphal/transport/can/transport.hpp

```diff
@@ -226,6 +237,11 @@ class TransportImpl final : public ICanTransport, private TransportDelegate  //
                               CANARD_NODE_ID_MAX + 1};
     }
 
+    void setTransientErrorHandler(TransientErrorHandler handler) override
```

Comment:
Proper docs will come in the next PR when there will be more experience. For now this is just blueprint.